build/repo.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+   <ItemGroup>
+     <ExcludeFromPack Include="$(RepositoryRoot)src\Microsoft.AspNetCore.AzureAppServices.TestBundle\Microsoft.AspNetCore.AzureAppServices.TestBundle.csproj" />
```

Comment:
This one's an odd duck. It doesn't follow the patterns of src or test. Maybe something like `packaging/TestBundle/`?